### PR TITLE
Draft: Log level improvement

### PR DESCRIPTION
**Pull request by Stojek139808**

### CODE REVIEW

File: src/soc/ibm/power9/istep_13_11.c [hunk 1]
```diff
@@ -145,7 +145,7 @@ static void dump_cal_errors(uint8_t chip, int mcs_i, int mca_i)
 		printk(BIOS_ERR, "DP %d\n", dp);
 		printk(BIOS_ERR, "\t%#16.16llx - RD_VREF_CAL_ERROR\n",
 		       dp_mca_read(chip, id, dp, mca_i, DDRPHY_DP16_RD_VREF_CAL_ERROR_P0_0));
-		printk(BIOS_ERR, "\t%#16.16llx - DQ_BIT_DISABLE_RP0\n",
+		printk(RAM_DEBUG, "\t%#16.16llx - DQ_BIT_DISABLE_RP0\n",
```
Comment (krystian-hebel) [inline src/soc/ibm/power9/istep_13_11.c:148]: Please change all of lines printed by this function to `RAM_SPEW` and remove `#if CONFIG(DEBUG_RAM_SETUP)`

File: src/soc/ibm/power9/istep_13_3.c [hunk 2]
```diff
@@ -115,7 +115,7 @@ static void mem_pll_initf(uint8_t chip)
 
 		/* This may depend on the requested frequency, but for current setup in our
 		 * lab this is ~3ms both for coreboot and Hostboot. */
-		printk(BIOS_EMERG, "putRing took %ld ms\n", time);
+		printk(BIOS_INFO, "putRing took %ld ms\n", time);
```
Comment (krystian-hebel) [inline src/soc/ibm/power9/istep_13_3.c:118]: `BIOS_DEBUG`, this is short enough

File: src/soc/ibm/power9/istep_13_8.c [hunk 3]
```diff
@@ -596,7 +596,7 @@ static void thermal_throttle_scominit(uint8_t chip, int mcs_i, int mca_i)
 	    // Set to disable permanently due to hardware design bug (HW403028) that won't be changed
 	    [53]    MBA_FARB3Q_CFG_NM_CHANGE_AFTER_SYNC = 0
 	*/
-	printk(BIOS_EMERG, "Please FIXME: ATTR_MSS_RUNTIME_MEM_THROTTLED_N_COMMANDS_PER_SLOT\n");
+	printk(BIOS_DEBUG, "Please FIXME: ATTR_MSS_RUNTIME_MEM_THROTTLED_N_COMMANDS_PER_SLOT\n");
```
Comment (krystian-hebel) [inline src/soc/ibm/power9/istep_13_8.c:599]: This was recently changed, you may want to amend the commit and skip it so there will be no conflicts during rebase.

File: src/soc/ibm/power9/romstage.c [hunk 4]
```diff
@@ -363,7 +363,7 @@ static void build_mvpds(uint8_t chips)
 {
 	uint8_t chip;
 
-	printk(BIOS_EMERG, "Building MVPDs...\n");
+	printk(BIOS_INFO, "Building MVPDs...\n");
```
Comment (krystian-hebel) [inline src/soc/ibm/power9/romstage.c:366]: `BIOS_NOTICE`, this takes very long time

File: src/soc/ibm/power9/istep_13_11.c (hunk 1 above)
Comment (Stojek139808) [reply src/soc/ibm/power9/istep_13_11.c]: Zrobiłem. Jeszcze ew. w ccs.c jest taki sam blok, też go tak zmodyfikować?

File: src/soc/ibm/power9/istep_13_3.c (hunk 2 above)
Comment (Stojek139808) [reply src/soc/ibm/power9/istep_13_3.c]: Zmienione

File: src/soc/ibm/power9/romstage.c (hunk 4 above)
Comment (Stojek139808) [reply src/soc/ibm/power9/romstage.c]: Ok, pushed a change with this.

**Comment by Stojek139808** [comment]
I've rebased onto raptor-cs_talos-2/develop to solve conflicts. Coreboot was compiled and ran before pushing, no issues there.

**Review by SergiiDmytruk — APPROVED** [review]
`ccs.c` wasn't compiling and some prints weren't updated (primarily in `HOMER`). Fixed that as well as commit structure and added `Signed-off-by` and `Change-Id`.

**Pull request merged**